packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -14,11 +15,11 @@
     max-height: none;
     padding: 0;
     margin: 0;
-    overflow-x: hidden;
-    overflow-y: auto;
+    overflow-x: hidden; // DEPRECATED, to be removed in next major version.
```

Comment:
Please, take a look at https://github.com/lmc-eu/spirit-design-system/pull/643/files#diff-1b2b57c18480e838c2d0a15737a414f42f72252939af0b49a10544bd0057baa5 and use the same comment. Will be spotted by lint and logged into the user console.

Reply:
OK, changed everywhere. I also simplified the code so the deprecated parts are either to be renamed or deleted, so no refactoring will be necessary (I also tested this scenario).